packages/node_modules/nav-frontend-skjema/src/fnr-input.tsx

```diff
@@ -24,7 +24,9 @@ class FnrInput extends React.Component<FnrInputProps> {
         return (
             <Input
                 {...this.props}
-                type="number"
+                type="text"
+                inputMode="numeric"
+                pattern="[0-9]*"
```

Comment:
Hm, tror dette potensielt kan utgjøre en breaking change. Bør muligens bumpe `major` her også for å være litt på den sikre siden? 🤔

Reply:
Sant, er ikke sikkert alle brukere parser inputtet de får riktig.